### PR TITLE
Css search

### PR DESCRIPTION
**Pull request by remy-rm**
add css and reesponsive for all page

### CODE REVIEW

**Review by ArPimentel — APPROVED** [review]

**Review by ArPimentel — DISMISSED** [review]

File: src/app/component/create-user-modal/create-user-modal.component.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  styleUrls: ['./create-user-modal.component.scss'],
+})
+export class CreateUserModalComponent implements OnInit {
+  isCreateModal = false;
+  userForm: any = {
+    firstname: '',
+    email: '',
+    password: '',
+    confirmPassword: '',
+    // Variable pour stocker l'e-mail de réinitialisation du mot de passe
```
Comment (Benja-san) [inline src/app/component/create-user-modal/create-user-modal.component.ts:19]: dead code

File: src/app/component/create-user-modal/create-user-modal.component.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+  isValidEmail(email: string): boolean {
+    return true;
+  }
+
+  ngOnInit(): void {
+    this.isCreateModal = this.authUserService.isCreatedIn();
+  }
+
+  someFunction() {
```
Comment (Benja-san) [inline src/app/component/create-user-modal/create-user-modal.component.ts:44]: what is this name??

File: src/app/component/login-modal/login-modal.component.ts [hunk 3]
```diff
@@ -1,36 +1,56 @@
...
+    this.loginSuccessSubscription = new Subscription();
+  }
+
+  showCreateModal(): void {
+    this.modalService.show(CreateUserModalComponent, {
+      initialState: {},
+    });
+  }
+
+  someFunction() {
```
Comment (Benja-san) [inline src/app/component/login-modal/login-modal.component.ts:34]: change function name

File: src/app/component/login-modal/login-modal.component.ts [hunk 4]
```diff
@@ -1,36 +1,56 @@
...
   ngOnInit(): void {
     this.isLoggedIn = this.authUserService.isLoggedIn();
   }
-
+  ngOnDestroy(): void {
+    this.loginSuccessSubscription.unsubscribe();
+  }
   onSubmit(): void {
     this.authUserService.login(this.email, this.password).subscribe(
       (response: any) => {
```
Comment (Benja-san) [inline src/app/component/login-modal/login-modal.component.ts:46]: remove any type

File: src/app/component/navbar/navbar.component.ts [hunk 5]
```diff
@@ -120,6 +122,7 @@ export class NavbarComponent implements OnInit, OnDestroy {
       this.countriesList.push(countries);
     }
     this.searchRecipes();
+    console.log(this.countriesList);
```
Comment (Benja-san) [inline src/app/component/navbar/navbar.component.ts:125]: remove all useless console logs

File: src/app/services/auth-user/auth-user.service.ts [hunk 6]
```diff
@@ -25,6 +25,23 @@ export class AuthUserService {
     private tokenService: TokenService
   ) {}
 
+  // Define the showResetPasswordForm method
```
Comment (Benja-san) [inline src/app/services/auth-user/auth-user.service.ts:28]: remove dead code

File: src/app/services/auth-user/auth-user.service.ts [hunk 7]
```diff
@@ -25,6 +25,23 @@ export class AuthUserService {
...
+  showResetPasswordForm(): void {
+    // Implementation here...
+  }
+
+  registerUser(email: string, password: string, firstname: string): Observable<any> {
+    return this.http.post<any>(`${this.baseUrl}/users/create`, { email, password, firstname });
```
Comment (Benja-san) [inline src/app/services/auth-user/auth-user.service.ts:34]: remove any type

File: src/app/services/auth-user/auth-user.service.ts [hunk 8]
```diff
@@ -25,6 +25,23 @@ export class AuthUserService {
...
+  }
+
+  sendPasswordResetEmail(email: string): Observable<any> {
```
Comment (Benja-san) [inline src/app/services/auth-user/auth-user.service.ts:37]: remove any type

File: src/app/services/auth-user/auth-user.service.ts [hunk 9]
```diff
@@ -25,6 +25,23 @@ export class AuthUserService {
...
+    return this.http.post<any>(`${this.baseUrl}/auth/reset-password`, { email });
+  }
+
+  resetPassword(email: string): Observable<any> {
```
Comment (Benja-san) [inline src/app/services/auth-user/auth-user.service.ts:41]: remove any type

**Review by Benja-san — CHANGES_REQUESTED** [review]

**Review by Benja-san — APPROVED** [review]

**Review by ArPimentel — APPROVED** [review]

**Pull request merged**